lib/protobuf/generators/field_generator.rb

```diff
@@ -20,7 +20,7 @@ class FieldGenerator < Base
       attr_reader :field_options
 
       def applicable_options
-        @applicable_options ||= field_options.map { |k, v| ":#{k} => #{v}" }
```

Comment:
it's for extension names with a dot in the middle -- wouldn't be valid ruby syntax with `:foo.bar` so it has to be `:"foo.bar"`